softwareComponents/simplesimClientLib/include/simplesim_client.hpp

```diff
@@ -173,18 +192,25 @@ class SimplesimClient
         return settingsCmd;
     }
 
+    std::unique_ptr< Ui::SimplesimClient > _ui;
```

Comment:
@xondika All member variables should be either all public or all private.
Also keep the code style at least in one class (underscore in front of variable here).

Already fixed